testdata/testdata.go

```diff
@@ -245,3 +245,122 @@ func GenTestData(mem memory.Allocator, sc *arrow.Schema, opts GenTestDataOptions
 	}
 	return records
 }
+
```

Comment:
Can you also put a comment that it is deprecated and will be removed in the future?

Reply:
Lint will fail the deprecation notice then.

Reply:
done in https://github.com/cloudquery/plugin-sdk/pull/812